### ISSUE TITLE
Tablet: MC-Auswahl wird beim ersten Auswahltippen in einer Unit nicht angenommen

### ISSUE THREAD

**Issue by jkoelm** [iqb-berlin/verona-modules-aspect#252]
v1.30/1.24

Am PC funktioniert die Auswahl einer MC-Option normal. Am Tablet ist es aber so, dass beim Betreten einer neuen Unit die erste Auswahl, die ich treffe, nicht angenommen wird. Ich tippe also auf die MC-Option und nichts passiert. Beim zweiten Tippen wird die Auswahl angenommen und danach innerhalb der Unit immer normal beim ersten Tippen. Gehe ich eine Unit zurück und wieder vor, wird wieder die erste Auswahl nicht genommen.

**Comment by jojohoch** [comment]
Auf welchem Tablet tritt der Bug auf? Auf iPad mit iOS 15.3. lässt sich der Bug weder lokal noch im Teststudio reproduzieren. Tritt er evtl. nur im Testcenter auf?

**Comment by jkoelm** [comment]
Ja, uns ist es im Testcenter aufgefallen. Im Teststudio habe ich es gerade noch mal ausprobiert, da funktoiniert es normal. Wir haben mit den IQB-Tablets getestet (iOS 15.3.1)

**Comment by jojohoch** [comment]
Scheint mir ein Testcenter IFrame-Focus-Problem zu sein

**Comment by jojohoch** [comment]
Ich habe ein Testpaket mit einer Unit bestehend aus mehreren MCs im Terststudio angelegt und dieses ins Testcenter importiert. Nach dem Öffnen des Testhefts konnte ich wie gewollt und ohne Fehlverhalten, eine Option bei der ersten Berührung auswählen.
-> Vermutlich war das beobachete Verhalten ein Bedienungsfehler. Nasse Finger, lange Fingernägel, ungenaues oder verwischtes Tippen.

**Comment by jkoelm** [comment]
Nein, einen Bedienungsfehler können wir ausschließen, Pauline und ich haben es viele Male ausprobiert. Wir schicken dir mal Testcenter-Zugänge über einen anderen Kanal.
